data/beps_completed.yml

```diff
@@ -123,6 +134,7 @@
 
 -   number: '027'
     title: BIDS Applications 2.0
+    link: https://docs.google.com/document/d/104HLZedFtx0TaXEUwd7eyWvJUlc0CcSUtCzwjNgmGxE/
```

Comment:
link to the bids-execution website

this one too will be merged with the rest but once again let's use valid URLs for now